src/main/java/net/mcreator/ui/MCreator.java

```diff
@@ -296,7 +297,8 @@ else if (screenSize.getWidth() > 1290 && screenSize.getHeight() > 795)
 			// if we need to setup the workspace, we do so
 			if (WorkspaceGeneratorSetup.shouldSetupBeRan(workspace.getGenerator())) {
 				WorkspaceGeneratorSetupDialog.runSetup(this,
-						PreferencesManager.PREFERENCES.notifications.openWhatsNextPage);
+						PreferencesManager.PREFERENCES.notifications.openWhatsNextPage

```

Comment:
As this is for devs, it would be better to check MCreatorVersionNumber#isDevelopment instead